packages/frontend/src/creation-form/components/oracles-configuration/oracle-creation-form/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { ErrorFeedback, Loader } from "@carrot-kpi/ui";
+import { Template } from "@carrot-kpi/sdk";
+import { OracleCreationFormWrapper } from "../oracle-creation-form-wrapper";
+import {
+    NamespacedTranslateFunction,
+    OracleCreationFormProps,
+    OracleInitializationBundleGetter,
+    useResolvedTemplate,
+} from "@carrot-kpi/react";
+import { AugmentedOracleDataMap } from "..";
+
+interface OraclesAccordionProps {
```

Comment:
This should be `OracleCreationFormProps` I guess.

Reply:
Addressed in bd61c1a740b3c01f9150fa350084963893855b0e